samples/react/ElmishCalc/src/State.fs

```diff
@@ -0,0 +1,77 @@
+module Calc.State
+
+open Calc.Types
+
+/// Active pattern that matches with an operation
+let (|Operation|_|) = function 
```

Comment:
I am using it a lot inside the `update` function, so instead of checking the four operations every time, I check them all at once with it